### DISCUSSION TITLE
How to make a core-shell structure in neper?

### DISCUSSION THREAD

**Discussion by siwux** [neperfepx/neper#292] [Q&A] [answered]
I want to build a quadratic unit model in neper. Each cell is similar to the core-shell model

**Comment by rquey** [comment, marked as answer]
This is probably not possible, but a scheme would help.

**Comment by siwux** [comment]
Thank you for your answer




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "rquey/neper"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月20日(星期三) 下午5:39
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [rquey/neper] How to make a core-shell structure in neper? (Discussion #292)





 
This is probably not possible, but a scheme would help.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
